src/current/v23.1/connection-pooling.md

```diff
@@ -29,34 +29,90 @@ Connection pooling can be a enabled as a feature of the driver, a separate libra
 
 Storage and network performance also will affect the ability of a thread to fully execute. If a thread is blocked by network or storage latency, adding connections to the pool is a good idea so other threads can execute while the original thread is being blocked.
 
 Cockroach Labs performed lab testing of various customer workloads and found no improvement in scalability beyond:
 
-**connections = (number of cores * 4)**
+**active connections = (number of cores * 4)**
 
 Many workloads perform best when the maximum number of active connections is between 2 and 4 times the number of CPU cores in the cluster.
 
 If you have a large number of services connecting to the same cluster, make sure the number of concurrent active connections across all the services does not exceed this recommendation by a large amount. If each service has its own connection pool, then you should make sure the sum of all the pool sizes is close to our maximum connections recommendation. Each workload and application is different, so you should conduct testing to determine the best-performing pool sizes for each service in your architecture.
 
 In addition to setting a maximum connection pool size, set the maximum number of idle connections if possible. Cockroach Labs recommends setting the maximum number of idle connections to the maximum pool size. While this uses more memory, it allows many connections when concurrency is high without having to create a new connection for every new operation.
 
+Configure the minimum number of connections to equal to the maximum number of connections, creating a fixed pool size.
+
+Set the maximum connection timeout value to 300 seconds, or 5 minutes. Do not set the connection pool timeout values to be too short, as it may cause the connection pool software to close and reopen connections frequently, causing increased latency. [Monitor the `sql.new_conns` metric](#monitor-new-connections) to make sure the timeout values are set correctly.
+
 {% include {{page.version.version}}/sql/server-side-connection-limit.md %} This may be useful in addition to your connection pool settings.
 
-## Validating connections in a pool
+</section>
+
+### Size connection pools in multi-region clusters
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+Similar to single-region {{ site.data.products.serverless }} clusters, the purpose of connection pools in multi-region {{ site.data.products.serverless }} clusters is to reduce connection latency. Add a connection pool for each region, and size the idle pool for your application and workload.
```

Comment:
I don't see a need to create a connection pool for each region. That's b/c we have a very strong recommendation that an app worker would run in the same region(s) as the database, and then only connect to its local region. That is, an app worker running in `us-east-1` would only connect to a DB node running in `us-east-1`. It should not be connecting to remote regions. We do not want to dispense any advice that might make customers think we recommend doing this.